src/ui/List/DefaultList.stories.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
-import { List } from 'semantic-ui-react';
+import { List, Container } from 'semantic-ui-react';
 
 export default {
   title: 'Components/List',
   component: List,
   argTypes: {},
 };
 
-const Template = (args) => <List {...args}></List>;
+const Template = (args) => (
+  <Container>
+    <List {...args}></List>
```

Comment:
@g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<ul>
  <li>
```

Reply:
@avoinea To avoid overrides we are using dl/dt in the updated files.